COPYRIGHT

```diff
@@ -0,0 +1,5 @@
+Copyright 2022 Adobe. All rights reserved.
```

Comment:
Why the year in copyrights is 2022?

Reply:
Copy paste and missed to update it. Will change it to 2024